src/SwedbankPay.Sdk/Identifiable.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+
+namespace SwedbankPay.Sdk
+{
+    /// <summary>
+    /// Contains a <seealso cref="Uri"/> to a uniquely identifiable resource.
+    /// </summary>
+    public class Identifiable

```

Comment:
Would it be a good idea to add an interface for `Identifiable` as well?

```suggestion
    public class Identifiable : IIdentifiable
```